dds/DCPS/DomainParticipantImpl.cpp

```diff
@@ -765,17 +764,17 @@ DDS::ReturnCode_t DomainParticipantImpl::delete_contentfilteredtopic(
     if (DCPS_debug_level > 3) {
       ACE_ERROR((LM_ERROR, ACE_TEXT("(%P|%t) ERROR: ")
         ACE_TEXT("DomainParticipantImpl::delete_contentfilteredtopic, ")
-        ACE_TEXT("can't delete a content-filtered topic \"%C\" ")
+        ACE_TEXT("can't delete a contentfiltered topic \"%C\" ")
```

Comment:
Please check the changes that are being made to log message text on this line and in the block below.  My previous comment was that we should just use the log message text as it existed before this PR.  Also the log message that starts on 853 would then need to change for consistency.